specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+This is the specified format for providing data in the form of key-value pairs, which are also known as name-value pairs, attribute-value pairs, and field-value pairs in other contexts. 
+
+This format is often used in place of enumerating data which would be inherently sparse and/or without predetermined keys (attributes) into separate columns. To faciliate parsing, this format does not support nesting.
+
```

Comment:
From your initial paragraphs, tried to a bit less technical.  Feel free to judge as you wish ...

The Key-Value Format specifies the format for providing data as key-value pairs, also known as name-value pairs, attribute-value pairs, and field-value pairs in other contexts.

The Key-Value Format mirrors JSON's ECMA-404 with a handful or extra requirements and in most cases, more efficiently consolidates column-based data than the CSV format.

Reply:
Also removed this line "To faciliate parsing, this format does not support nesting." because it seemed duplicated in the requirements below

Reply:
@cnharris10 Not seeing where this is mentioned in requirements

Reply:
Referring to this line, `* Values MUST NOT be an object nor an array.`.  If this is true, nesting is not possible.